network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta.html

```diff
@@ -1,5 +1,3 @@
-{{ cta.type }}
```

Comment:
This doesn't do anything. I'm guessing someone had this for testing purpose but forgot to remove it before merging the PR.